6.bulk_Morphology_Elastic_Network/0.split_data/notebooks/1.get_cytokine_list.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
<p>Minimalistic code, I like it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>